api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserModel.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "users")
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@Setter(value = AccessLevel.PACKAGE)
+@Getter
+public class UserModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String twitchUsername;
+
+    private Timestamp createTimestamp;
```

Comment:
Super minor, Ive seen createdAt used pretty commonly. CreateTimestamp is obviously fine, but I think the industry standard is to use createdAt, updatedAt, ect...

Reply:
Lol "industry standard" might be a bit strong of a description. Most schemas I've seen tend to vary greatly (databases can't even agree on a common syntax, let alone a standard way of naming variables inside of a schema).

I can make the change.

Reply:
up to you. it was super minor. 

In hindsight you're right. Industry standard was 100% too strong.

naming standard across 4 dbs and 3 companies is accurate.